change/@uifabric-react-cards-2019-07-11-15-35-18.json

```diff
@@ -0,0 +1,8 @@
+{
+  "comment": "Card: Importing from office-ui-fabric-react directly instead of from office-ui-fabric-react/lib/...",
```

Comment:
?

Reply:
Just a precommit hook that auto converts these stray change files from rush format to beachball.